server/controller/itemController.ts

```diff
@@ -0,0 +1,53 @@
+import { Request,Response} from 'express';
+import ItemService from '../service/itemService';
+
+class ItemController {
```

Comment:
I don't understand what is item, is it means ticket, if so we can name it TicketController